src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.html

```diff
@@ -53,7 +53,7 @@ <h4 class="modal-title">Copy Questions</h4>
                   </thead>
                   <tbody>
                     <tr *ngFor="let questionCandidate of feedbackSessionTabModel.questionsTableRowModels">
-                      <td><input type="checkbox" class="form-control" [(ngModel)]="questionCandidate.isSelected"></td>
+                      <td><input type="checkbox" class="form-controller" [(ngModel)]="questionCandidate.isSelected"></td>
```

Comment:
Can remove the `form-controller` class here as the css for it has been removed! So it doesn't affect the element anymore